lib/oban/telemetry.ex

```diff
@@ -312,7 +312,8 @@ defmodule Oban.Telemetry do
 
   #### Queue Shutdown Events
 
-  * `ellapsed` — the amount of time the queue waited for shutdown, in milliseconds
+  * `elapsed` — the amount of time the queue waited for shutdown, in milliseconds
+  * `ellapsed` — alias for elapsed
```

Comment:
What is the best way to preserve backwards compatibility?

Reply:
There's no need to maintain compatibility for the log message. We can just fix the spelling.